var/spack/repos/builtin/packages/mesa/package.py

```diff
@@ -106,7 +153,31 @@ def configure_args(self):
                 args.append('--disable-llvm-shared-libs')
             args.append('--with-llvm-prefix=%s' % spec['llvm'].prefix)
 
-        if drivers:
-            args.append('--with-gallium-drivers=' + ','.join(drivers))
+        args.append('--with-gallium-drivers=' + ','.join(drivers))
+
+        # Avoid errors due to missing clock_gettime symbol:
+        arch = spec.architecture
+        if arch.platform == 'linux':
+            args.extend(['LIBS=-lrt'])
 
         return args
+
+    def configure(self, spec, prefix):
+        """Configure mesa, detecting if libsysfs is required
+        for DRI support on the build host.
+        """
+        options = ['--prefix={0}'.format(prefix)] + self.configure_args()
+        configure = inspect.getmodule(self).configure
```

Comment:
You don't need this line (or the inspect import). `configure` should already be in your module scope.